src/pages/_app.tsx

```diff
@@ -8,7 +8,7 @@ import firebase from "../../utils/firebase/firebase";
 
 const httpLink = new HttpLink({ uri: process.env.NEXT_PUBLIC_API_URL });
 
-const authLink = setContext(async (_, { headers }) => {
+const authLink = setContext(async (_, { headers, ...context }) => {
```

Comment:
Just to clarify the idea for myself: links are sort of like middleware that graphql requests go through before being sent to the server?

Reply:
Side note: Do you think adding an onError link like in the [introductions page to links](https://www.apollographql.com/docs/react/api/link/introduction/) would be a good idea?

Reply:
Probably unnecessary since we don't have any universal actions to perform on errors.